.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
       - run:
           name: Concatenate all package-lock.json files into single file for checksum
-          command: cat package-lock.json api/package-lock.json authorizer/package-lock.json > combined-package-lock.txt
+          command: cat package-lock.json api/package-lock.json > combined-package-lock.txt
```

Comment:
Just interested: does this work just `cat`ing them together (appreciate we were already doing it)? 😕 I would expect the result is invalid JSON.

Reply:
Yeah, the output is just used to generate a checksum which is used as a cache key

Reply:
Ah cool yeah I see now, hadn't looked at how it was used. Thanks!